common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -0,0 +1,651 @@
+                    minY = overlay.getRenderY();
+                    maxY = overlay.getRenderY() + overlay.getHeight();
+                }
+                case Right -> {
+                    minX = overlay.getRenderX() + overlay.getWidth() - MAX_CLICK_DISTANCE / 2f;
+                    maxX = overlay.getRenderX() + overlay.getWidth() + MAX_CLICK_DISTANCE / 2f;
+                    minY = overlay.getRenderY();
+                    maxY = overlay.getRenderY() + overlay.getHeight();
+                }
+                case Bottom -> {
+                    minX = overlay.getRenderX();
+                    maxX = overlay.getRenderX() + overlay.getWidth();
+                    minY = overlay.getRenderY() + overlay.getHeight() - MAX_CLICK_DISTANCE / 2f;
+                    maxY = overlay.getRenderY() + overlay.getHeight() + MAX_CLICK_DISTANCE / 2f;
+                }
+                default -> {
+                    // should not happen
+                    continue;
+                }
+            }
+
+            if (mouseX >= minX && mouseX <= maxX && mouseY >= minY && mouseY <= maxY) {
+                selectedEdge = value;
+                selectionMode = SelectionMode.Edge;
+
+                return false;
+            }
+        }
+
+        if (isMouseHoveringOverlay(selectedOverlay, mouseX, mouseY)) {
+            selectionMode = SelectionMode.Area;
+
+            return false;
+        }
+
+        return super.mouseClicked(mouseX, mouseY, button);
+    }
+
+    @Override
+    public boolean mouseReleased(double mouseX, double mouseY, int button) {
+        if (testMode) return false;
+
+        resetSelection();
+        return super.mouseReleased(mouseX, mouseY, button);
+    }
+
+    @Override
+    public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
+        if (testMode) return false;
+
+        if (keyCode == GLFW.GLFW_KEY_UP || keyCode == GLFW.GLFW_KEY_DOWN) {
+            int index = selectedOverlay.getRenderVerticalAlignment().ordinal();
+
+            if (keyCode == GLFW.GLFW_KEY_DOWN) {
+                index += 1;
+            } else {
+                index -= 1;
+            }
+
+            VerticalAlignment[] values = VerticalAlignment.values();
+            index = (values.length + index) % values.length;
+            selectedOverlay.setVerticalAlignmentOverride(values[index]);
+            saveConfigForOverlay();
+        } else if (keyCode == GLFW.GLFW_KEY_RIGHT || keyCode == GLFW.GLFW_KEY_LEFT) {
+            int index = selectedOverlay.getRenderHorizontalAlignment().ordinal();
+
+            if (keyCode == GLFW.GLFW_KEY_RIGHT) {
+                index += 1;
+            } else {
+                index -= 1;
+            }
+
+            HorizontalAlignment[] values = HorizontalAlignment.values();
+            index = (values.length + index) % values.length;
+            selectedOverlay.setHorizontalAlignmentOverride(values[index]);
+            saveConfigForOverlay();
```

Comment:
Does this gets saved even if we do not press Apply?

Reply:
I think it does save position if you move things around.

Reply:
I need to verify this. If you just press close, and not apply, it should neither save a new placement or size of the overlay, nor the text alignment inside the overlay.

Reply:
I just fixed this. Thanks for pointing out.